code/scripting/api/libs/ui.cpp

```diff
@@ -491,5 +492,42 @@ ADE_FUNC(runBriefingStageHook, l_UserInterface_CmdBrief, "number oldStage, numbe
 	return ADE_RETURN_NIL;
 }
 
+//**********SUBLIBRARY: UserInterface/LoopBrief
+ADE_LIB_DERIV(l_UserInterface_LoopBrief,
+	"LoopBrief",
+	nullptr,
+	"API for accessing data related to the loop brief UI.<br><b>Warning:</b> This is an internal "
+	"API for the new UI system. This should not be used by other code and may be removed in the future!",
+	l_UserInterface);
+
+ADE_FUNC(getLoopBrief,
+	l_UserInterface_LoopBrief,
+	nullptr,
+	"Get the loop brief.",
+	"loop_brief_stage",
+	"The loop brief data")
+{
+	return ade_set_args(L, "o", l_LoopBriefStage.Set(Campaign.missions[Campaign.current_mission]));
+}
+
+ADE_FUNC(setLoopChoice, l_UserInterface_LoopBrief, "boolean", "Accepts mission outcome and then True to go to loop, False to skip", nullptr, nullptr)
+{
+	bool choice = false;
+	ade_get_args(L, "b", &choice);
```

Comment:
If this is intended to be a required parameter, then there should be an `if (!ade_get_args...)` check with an appropriate return value.

Alternatively, this could be considered an optional parameter, defaulting to the value of `choice`.  In this case, an `if()` check is unnecessary, but the parameter string should be "|b" to indicate that the boolean is optional.